src/dune_engine/cache_update_result.ml

```diff
@@ -0,0 +1,28 @@
+open Import
```

Comment:
How about renaming this module to `fs_cache_update_result`? Seems a bit more precise.

Reply:
Sounds fine with me.